libraries/I2S/src/I2S.cpp

```diff
@@ -145,9 +147,15 @@ int I2SClass::begin(int mode, long sampleRate, int bitsPerSample, bool driveCloc
   i2sd.setSlotSize(_deviceIndex, bitsPerSample);
   i2sd.setDataSize(_deviceIndex, bitsPerSample);
 
+#if defined(__SAMD51__)
+  pinPeripheral(_sckPin, PIO_I2S);
+  pinPeripheral(_fsPin,  PIO_I2S);
```

Comment:
As written, I get the bit sync clock (sampleRate * 2 * bitsPerSample) on the pin identified by `_sckPin` and the _master clock_ (sampleRate * 256) on the pin identified by `_fsPin`. And with my mic connected to these pins nothing comes out of the mic's data pin. However, if I add another statement after these two:
```c++
pinPeripheral(10, PIO_I2S); // just used pin 10 for testing
```
I then in addition to the get the frame sync clock (sampleRate) on the new pin (pin 10 in this case). And if I wire up the new pin to my mic instead of the old pin (`_fsPin`), I get data out of my mic's data pin.

Despite seeing data coming out of the mic on the scope, I'm still getting all zeros reading it in my sketch, but this is progress of a kind.